docs/docs/models/species-detection.md

```diff
@@ -262,6 +302,8 @@ video_loader_config:
     confidence: 0.25
     fill_mode: score_sorted
     n_frames: 32
+    image_height: 416
+    image_width: 416
```

Comment:
Update to new frame selection size (and maybe update default slowfast config?)

Reply:
This docs change brings the default in line with what is actually used. Whether what is used is desirable is another question.

The issue is that the official configs get generated from what was used for training, which in this case, was 416 x 416. If we change the default slowfast config manually, this will get overwritten the next time we publish models (since the official configs are generated from the full train configuration used for training, with the machine specific params removed). If we want to use slowfast with a different frame selection size, we'll need to change that workflow. 

I'm not super inclined to fix this as part of this PR which just aims to release the BNB model. It's still an open question if we continue to support the slowfast model and if we retrain it on more data, which could make this point moot.